src/styles/_colours.scss

```diff
@@ -6,18 +6,18 @@
 @import "./settings";
 
 // PRIMARY COLOURS
-$colour-dark-blue: #004864;
-$colour-uniprot-blue: #006e9a;
-$colour-medium-blue: #3ca3ca;
-$colour-light-blue: #93d4ee;
+$colour-dark-blue: #014371;
```

Comment:
These need to be renamed to the new names defined by Austra. Also update the colours showcase page.

Reply:
It's done!